hstest/environment/page.ts

```diff
@@ -13,12 +13,14 @@ class Page {
     isOpened: boolean;
     pageInstance!: puppeteer.Page;
     gotoOptions: puppeteer.WaitForOptions;
+    requests: Array<any>;
```

Comment:
```suggestion
    requests: Array<HTTPRequest>;
```
https://pptr.dev/api/puppeteer.httprequest/